src/uct/sm/mm/base/mm_iface.c

```diff
@@ -523,9 +523,9 @@ static UCS_CLASS_INIT_FUNC(uct_mm_iface_t, uct_md_h md, uct_worker_h worker,
 
     /* check the value defining the size of the FIFO element */
     if (mm_config->fifo_elem_size <= sizeof(uct_mm_fifo_element_t)) {
-        ucs_error("The UCT_MM_MAX_SHORT parameter must be larger than the FIFO "
-                  "element header size. ( > %ld bytes).",
-                  sizeof(uct_mm_fifo_element_t));
+        ucs_error("The "UCT_MM_CONFIG_FIFO_ELEM_SIZE" parameter (%u) must be "
```

Comment:
i'd just change name in this error output (also this define would print just a part of var name)

Reply:
thank you for the comment
done